.github/workflows/ci-post-commit-analyzer.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Setup ccache
-        uses: hendrikmuhs/ccache-action@v1
+        uses: hendrikmuhs/ccache-action@ed74d11c0b343532753ecead8a951bb09bb34bc9 # v1.2
```

Comment:
The ccache-action changes seem unrelated to the rest of the patch, can you split these out into a separate PR.

Reply:
They've been removed. I was just testing the changes on this PR. Sorry for the confusion.